server/routes/support/handlers/offenderDetail.ts

```diff
@@ -73,6 +88,47 @@ export default class OffenderDetailRoutes {
         pdu: probationPractitioner?.team?.borough?.description,
         region: probationPractitioner?.probationArea?.description,
       },
+      licence: await this.getLicenceDates(nomsId, user),
     })
   }
+
+  getLicenceDates = async (nomsId: string, user: User): Promise<LicenceDates> => {
+    const licenceSummary = await this.licenceServer.getLatestLicenceByNomisIdsAndStatus([nomsId], [], user)
+
+    if (!licenceSummary) {
+      return {
+        crd: 'Not found',
+        ard: 'Not found',
+        ssd: 'Not found',
+        sed: 'Not found',
+        led: 'Not found',
+        tussd: 'Not found',
+        tused: 'Not found',
+      }
+    }
+
+    const licence = await this.licenceServer.getLicence(licenceSummary.licenceId.toString(), user)
+
+    return {
+      crd: licence.conditionalReleaseDate
+        ? moment(licence.conditionalReleaseDate, 'DD/MM/YYYY').format('DD MMM YYYY')
```

Comment:
could extract out a helper function and tidy this up a bit?

```
ard: formatLicenceDate(licence.actualReleaseDate),
ssd: formatLicenceDate(licence.sentenceStartDate),
```

Reply:
Done.